gamemode/modules/drowning/sv_drowning.lua

```diff
@@ -8,35 +8,36 @@ util.AddNetworkString( "RVR_Player_Enter_Water" )
 util.AddNetworkString( "RVR_Player_Leave_Water" )
 util.AddNetworkString( "RVR_Player_Take_Drown_Damage" )
 
-local function isDrowning( player )
-    return ( CurTime() - player.FirstSubmergedTime ) >= config.DROWNING_THRESHOLD
+local function isDrowning( ply )
```

Comment:
What, you don't like `player`?

Reply:
Its an existing gmod library

Reply:
https://wiki.facepunch.com/gmod/player(library)